common/on_actions/imperia_on_monthtly_pulse.txt

```diff
@@ -129,6 +129,36 @@
 				multiplier = local_var:tmp_imperia_china_infamy_penalty_value
 			}
 		}
+		# Frankfurt Parliament variable cleanup
+		# Done here to maintain stability in inter-JE functions
+		if = {
+			limit = {
+				imperia_frankfurt_parliament_is_in_session = no
+				# At least one country has one of the following variables...
+				any_country = {
```

Comment:
Why do we check every country for every country? Can't we just check the current country only?

Reply:
Also why can't we clean up the variables when the journal entry becomes invalid/completes or whatever.

Reply:
The reason I'm not cleaning up per JE is that many of the variables are used down the line. Although I suppose I could split up the clean effect, so the JE-super specific variables gets cleaned.. 
But good point on the any_country in an every_country, I'll perform a fix for that